wdl/tasks/Sniffles2.wdl

```diff
@@ -0,0 +1,175 @@
+version 1.0
+
+
+import "Structs.wdl"
+
+
+workflow Sniffles2 {
+
+    meta {
+        description: "This workflow calls SV candidates using Sniffles2 population mode.
```

Comment:
are you sure if multi-line descriptions are allowed?
I'd love that to be true, but it may be not.